src/lamplib/src/genny/cli.py

```diff
@@ -430,6 +418,16 @@ def canaries(ctx: click.Context, canary_args: List[str]):
         "Not needed if you have built/installed these tools in hte --mongo-dir."
     ),
 )
+@click.option(
+    "--mongodb-commit",
+    type=str,
+    required=False,
+    default=None,
+    help=(
+        "The commit corresponding to the mongodb binary passed with -mongodb-archive-url option."
+        "Not needed if you are not passing --mongodb-archive-url."
```

Comment:
I'm confused, these lines say it has to correspond with the commit of the binary used with that option, but it shouldn't be used with that option? Is that contradictory?

Reply:
It says: - '--mongodb-commit' option is not needed if a user is not passing '--mongodb-archive-url'. 
It doesn't say it shouldn't be used with that option?

Reply:
removing this line to avoid confusion.